lib/stitches/generator_files/app/controllers/api/api_controller.rb

```diff
@@ -1,4 +1,11 @@
 class Api::ApiController < ActionController::Base
+  #
+  # API clients pass an API key instead of a CSRF token; Use
+  # :null_session CSRF protection to avoid an auth error for these
+  # clients.
+  #
+  protect_from_forgery with: :null_session
```

Comment:
I don't believe this will be necessary, now that [this change](https://github.com/stitchfix/stitches/pull/72/files#diff-c72d61bbc5ce40c6cff18d6d26d17d57R1) has been merged.